frontend/src/metabase-lib/mocks.ts

```diff
@@ -53,6 +53,8 @@ const SAVED_QUESTION = {
   name: "Q1",
   description: "",
   collection_id: null,
+  can_write: true,
+  result_metadata: ORDERS.fields.map(field => field.getPlainObject()),
```

Comment:
Can we make this an empty array by default? I think making all tests rely on the sample database fixture might get difficult to migrate from in the future.